client/grpc/tmservice/query.pb.go

```diff
@@ -9,7 +9,7 @@ import (
 	types "github.com/Finschia/finschia-sdk/codec/types"
 	query "github.com/Finschia/finschia-sdk/types/query"
 	_ "github.com/Finschia/ostracon/abci/types"
-	types2 "github.com/Finschia/ostracon/proto/ostracon/types"
+	types2 "github.com/tendermint/tendermint/proto/tendermint/types"
```

Comment:
This stub file is changed without changing proto file.

Reply:
I've fixed the corresponding .proto file and re-generated this.